cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1638,11 +1638,13 @@
     <value></value>
     <description>
       A comma-separated list of users for whom admin privileges are to be granted on the
-      CDAP instance during CDAP startup, so these these users can create namespaces, and
-      in effect bootstrap CDAP on an authorization-enabled cluster. The default is empty,
-      in which case no users have access to creating namespaces. In that scenario it may
-      be impossible to bootstrap CDAP, so it is recommended that at least one user be
-      provided as the administrative user.
+      CDAP instance during CDAP startup, so these these users can create namespaces. These
+      users are also granted admin privileges on the default namespace, so they can
+      manage privileges on the default namespace. This provides a way to bootstrap CDAP on
+      an authorization-enabled cluster. The default is empty, in which case no users have
+      access to creating namespaces or to manage privileges on the default namespace.
+      In that scenario it may CDAP has to be bootstrapped externally using interfaces
+      provided by the configured authorization extension.
```

Comment:
sounds good. Only, the last sentence is not really an option. So I'll just remove "it may mean that"